app/products.py

```diff
@@ -0,0 +1,114 @@
+"""
+ Must have the environment variables set properly prior to running the script
+"""
+from woocommerce import API
+import json
+import os
+from faker import Faker
+import random
+
+fake = Faker()
+
+
+def create_api_object():
+    wcapi = API(
+        url=os.environ.get('BASE_URL'),
+        consumer_key=os.environ.get('API_KEY'),
```

Comment:
I love that you put credentials as environment variable. But you are not doing any check if the variables are set.
So if the variables are not set you will get `None` and the API will fail for authentication. Then you will not know what is wrong. You will have to investigate to figure out that the env variables are not set.
So I like to make explicit check for the variable and if not set then raise an exception with a message right away.